config/deploy.rb

```diff
@@ -5,10 +5,12 @@
 set :repo_url, 'https://github.com/sul-dlss/frda.git'
 
 # Default branch is :master
-# ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+
+set :passenger_restart_with_touch, true
```

Comment:
Do we need `passenger_restart_with_touch`? I've had luck with the new-style way using `passenger-config`.